frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -548,9 +548,9 @@ local metadata_folder_help_text = _([[
-- all inside koreader/docsettings/: these sdr folders will only be visible and used by KOReader, and won't clutter your vision of your library directories with another file browser or from your computer. But any reorganisation of your library (directories or filename moves and renamings) may result in KOReader not finding your previous settings for these books. These settings won't be part of any synchronization or backups of your library.
-- all inside koreader/hashdocsettings/ as hashes: sdr folders are identified not by filepath and filename but by partial MD5 hash. This allows you to rename, move, and copy documents outside at will outside of KOReader, and KOReader will still link them to their metadata. However, any file modifications such as writing highlights into PDFs or downloading from Calibre may change the hash, resulting in metadata being unlinked from the file. Calculating file hashes may also impact loading and file browser browsing times. This option may be preferable for users with many documents that want to remove sdr folder clutter or who might have multiple copies of their documents in different directory paths or devices.]])
+- alongside the book file itself (the long time default): sdr folders will be visible when you browse your library directories with another file browser or from your computer, which may clutter your vision of your library. But this allows you to move them along when you reorganize your library, and also survives any renaming of parent directories. Also, if you perform directory synchronization or backups, your settings will be part of them.
+- all inside koreader/docsettings/: sdr folders will only be visible and used by KOReader, and won't clutter your vision of your library directories with another file browser or from your computer. But any reorganisation of your library (directories or filename moves and renamings) may result in KOReader not finding your previous settings for these books. These settings won't be part of any synchronization or backups of your library.
+- all inside koreader/hashdocsettings/ as hashes: sdr folders are identified not by filepath and filename but by partial MD5 hash, allowing you to rename, move, and copy documentst outside of KOReader without sdr folder clutter, while keeping them linked to their metadata. However, any file modifications such as writing highlights into PDFs or downloading from Calibre may change the hash, resulting in metadata being unlinked from the file. Calculating file hashes may also impact loading/file browsing times. This option may be preferable for users with multiple copies of their documents across different devices and directory paths.]])
```

Comment:
`documentst`

Reply:
For what it's worth. Two versions shortend by chatgpt using previous texts: 

In koreader/hashdocsettings/, files use MD5 hashes for identification, not file paths. This allows easy file management outside of KOReader while retaining metadata links. Note that modifying files may change the hash, potentially disconnecting metadata. Calculating hashes can impact loading and browsing times. This option suits users with many documents aiming to declutter sdr folders or managing copies across different locations/devices.


Inside koreader/hashdocsettings/, files are identified by partial MD5 hash, not by their path or name. This enables easy renaming, moving, and copying of documents outside of KOReader without cluttering sdr folders, while still maintaining metadata links. Note that file modifications like adding highlights or downloading from Calibre may alter the hash, potentially disconnecting metadata. Calculating hashes can affect loading and browsing times. This option suits users with copies across various devices and directories.

Reply:
Thanks, I used those for ideas to further shorten.